docs/jade-getting-started.md

```diff
@@ -367,6 +367,8 @@ export GOOGLE_ALLOWREUSEEXISTINGPROJECTS=true
 export AZURE_CREDENTIALS_HOMETENANTID=<value from the top of .github/workflows/int-and-connected-test-run.yml>
 export AZURE_CREDENTIALS_APPLICATIONID=<value from the top of .github/workflows/int-and-connected-test-run.yml>
 export AZURE_CREDENTIALS_SECRET=$(cat /tmp/jade-dev-azure.key)
+export AZURE_SYNAPSE_SQLADMINUSER=$(cat /tmp/jade-dev-synapse-admin.key)
```

Comment:
super nit: could this be `jade-dev-synapse-admin-user`?  That way it's symmetrical with `jade-dev-synapse-admin-password`